example-manifest.json

```diff
@@ -1,6 +1,6 @@
 {
   "baseFolderDownloads": [
-    "https://www.ripta.com/stuff/contentmgr/files/0/3cda81dfa140edbe9aae214b26245b4a/files/google_transit.zip",
+    "https://www.ripta.com/wp-content/uploads/2021/01/google_transit.zip",
     "https://download.geofabrik.de/north-america/us/rhode-island-latest.osm.pbf"
```

Comment:
In light of recent changes, these entries should be objects as described in README.

Reply:
Fixed in 7a9a956a6b31abb6b6607ed8ec8f73fef1d0fd76